### DISCUSSION TITLE
Help Moving Custom Filters Between Computers

### DISCUSSION THREAD

**Discussion by OsirisRaptor** [KSP-CKAN/CKAN#3772] [Q&A] [answered]
I spent a long time color organizing CKAN for my needs. I am trying to transfer the custom filters with CKAN but when I move the CKAN file directory from one computer to another my filters aren't retained. Can someone help?

**Comment by HebaruSan** [comment, marked as answer]
They're in `%LOCALAPPDATA%\CKAN\labels.json` on Windows or `~/.local/share/CKAN/labels.json` on Linux.

**Comment by OsirisRaptor** [comment]
Thank you very much.
